WalletWasabi/Blockchain/TransactionOutputs/CoinsView.cs

```diff
@@ -52,8 +47,6 @@ public bool TryGetByOutPoint(OutPoint outpoint, [NotNullWhen(true)] out SmartCoi
 
 	public Money TotalAmount() => Coins.Sum(x => x.Amount);
 
-	public SmartCoin[] ToArray() => Coins.ToArray();
-
 	public IEnumerator<SmartCoin> GetEnumerator() => Coins.GetEnumerator();
 
 	IEnumerator IEnumerable.GetEnumerator() => Coins.GetEnumerator();
```

Comment:
For me, `GetEnumerator()` is also unused.

Should we remove it or is there some external code which will call this?

Reply:
Don't waste time on it. I will check it out.